meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_2021.2.bbappend

```diff
@@ -2,5 +2,5 @@
 PROVIDES += "virtual/trusted-firmware-a"
 
 # Enable opteed as the main SPD provider (required for optee)
-EXTRA_OEMAKE:append:zynqmp = " SPD=opteed"
+EXTRA_OEMAKE:append:uz3eg-iocc = " SPD=opteed"
```

Comment:
Please change to 'uz' as we have other customer specific machines that are uz-based that would also be affected by this change.